editor/plugins/theme_editor_plugin.cpp

```diff
@@ -56,14 +56,10 @@ void ThemeEditor::_refresh_interval() {
 	_propagate_redraw(main_container);
 }
 
-void ThemeEditor::_type_menu_cbk(int p_option) {
-
-	type_edit->set_text(type_menu->get_popup()->get_item_text(p_option));
-}
 
```

Comment:
There are code style issues due to the two consecutive newlines here and where `_name_menu_cbk` was removed.

Reply:
And trailing whitespace: https://travis-ci.org/godotengine/godot/jobs/535167849

Reply:
forgot clang-format :P

Reply:
Ok now it should be fine.